src/Pages/CreatorPage.js

```diff
@@ -1,8 +1,8 @@
 import React, { useEffect, useState } from 'react';
```

Comment:
All of component names and folders should be capitalized! I think the current version in main has CreatorPage capitalized, so you had to make the change shown here. Can you make sure all the folders are capitalized and change the references as needed?